src/orion/core/worker/trial.py

```diff
@@ -192,8 +194,8 @@ def to_dict(self):
 
     def __str__(self):
         """Represent partially with a string."""
-        ret = "Trial(experiment={0}, status={1}, params.value={2})".format(
-            repr(self.experiment), repr(self._status), [p.value for p in self.params])
+        ret = "Trial(experiment={0}, status={1},\n      params={2})".format(
+            repr(self.experiment), repr(self._status), self.params_repr(sep=('\n' + ' ' * 13)))
```

Comment:
What about using `pprint` if we want some fancy `repr`?

Reply:
of course, but later